index.html

```diff
@@ -0,0 +1,64 @@
+<!-- landing page for personal portfolio -->
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title> Tamiko Terada </title>
+    <link rel="stylesheet" type="text/css" href="stylesheets/main.css">
+    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css">
+  </head>
+
+  <body>
+    <a href="html/menu.html" target="_parent"><button autofocus id="test"> </button></a>
+    <!-- header | name, image, banner, button-link -->
+    <header>
+      <div class="left-bar">
+        <p class="top-bar">Tamiko Terada</p>
+      </div>
+      <nav><a href="menu.html">menu</a></nav>
+    </header>
+
+    <div id="header-image"></div>
```

Comment:
Oh, never mind I see it in the css. A couple thoughts: 
- Something like `id="header-image-wrapper"` might be a more informative name to convey what this component is actually doing
- I think it's preferable to add image links via `<img>` tags so you can use their `alt` text